### ISSUE TITLE
Unable to select Legiones Hereticus on Mobile

### ISSUE THREAD

**Issue by Varistartled** [BSData/horus-heresy#2969]
When creating a new roster (or augmenting an existing one), the ability to select Legiones Hereticus does not appear on mobile.

1. Go to Battlescribe on Mobile
2. Download updated repository from github
3. Hit +
4. Select (HH V2) Horus Heresy (2022) v84
5. Select LA III: Emperor's Children v28
6. Select Crusade Force Organisation Detachment
7. Create roster by tapping the check mark
8. Scroll to Allegiance
9. Select Traitor
10. Option to select Legiones Hereticus does not appear on any subsequent or previous page

Once selecting traitor, an additional option to select hereticus should appear on the previous page. 

 - OS: Android
 - Battlescribe
 - App version 2.03.27
 - Data release version 17.4.0

**Comment by Mayegelt** [comment]
Select traitor allegiance first

**Comment by Th33z** [comment]
> Select traitor allegiance first

I was able to reproduce this issue on my Android phone (BS ver: 2.03.27, BS data version: Horus Heresy v17.4.0, Phone: Galaxy S22, Android OS 5.1). Selecting the Traitor allegiance should show the dropdown to select LA or Hereticus as described and how the desktop client works (which I can verify that portion works on desktop), but it is simply not present within the interface on mobile. See video attached for visibility. Let me know if you need any further info.

https://github.com/BSData/horus-heresy/assets/19887954/3653c911-1445-4b44-8c63-9218e55779d5

**Comment by Mayegelt** [comment]
I'm not sure why it's not working for you. Have you tried creating a new roster from scratch, or do a delete/reinstall in data

https://github.com/BSData/horus-heresy/assets/20280410/0fa6795b-71a4-4bea-8a26-c5108f6f7189

**Comment by Th33z** [comment]
> I'm not sure why it's not working for you. Have you tried creating a new roster from scratch, or do a delete/reinstall of the data files
> 
>  Screenrecorder-2023-11-03-07-36-58-842.mp4

Odd. When I tested this initially, I did go through and manually delete the data and reinstall, but it was still showing in the video.

I just tested it again, but also went through the effort of Deleting all data in my BS client (on the off chance it was still caching the previous repository, despite showing the current data version). After that, I wiped my cache on the app and rebooted, reinstalled v17.4.0, and it's now letting me select Hereticus. Curious thing, haven't had to do that before, but hey learning experiences I guess haha.
